src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMigrationIntTest.kt

```diff
@@ -225,6 +232,59 @@ class ContactPersonMigrationIntTest : SqsIntegrationTestBase() {
       }
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
setup will be run once so we can split suite to separate tests but maintain performance